firestore-incremental-capture/extension.yaml

```diff
@@ -113,19 +113,15 @@ resources:
       location: ${LOCATION}
       taskQueueTrigger: {}
 
-  - name: updateDataflowStatus
-    type: firebaseextensions.v1beta.v2function
+  - name: checkDataflowJobState
+    type: firebaseextensions.v1beta.function
     description: >-
-      Updates the status of the DataFlow job in Firestore.
+      Checks if the DataFlow job is done to update the status
     properties:
+      availableMemoryMb: 512
+      runtime: nodejs18
       location: ${LOCATION}
-      buildConfig:
-        runtime: nodejs18
-      eventTrigger:
-        eventType: google.cloud.dataflow.job.v1beta3.statusChanged
-        # eventFilters:
-        #   - attribute: job
-        #     value: projects/${param:PROJECT_ID}/locations/${LOCATION}/jobs/${param:EXT_INSTANCE_ID}-*
+      taskQueueTrigger: {}
```

Comment:
can we add a retry logic for all of the cloud tasks functions

Reply:
@pr-Mais added retry config for all cloud tasks